bloom-housing/bloom
# feat: adding new listing availability field

# Pull Request Template

## Issue Overview

This PR addresses [#issue](https://github.com/bloom-housing/bloom/issues/2716)


## Description
This removes the unit status from the unit form on the partner side

This also adds the new listing availability field on the partner side

## How Can This Be Tested/Reviewed?
Head to https://dev-partners-bloom.netlify.app
go to create (or edit) a new listing
notice that there is the new `What is the listing availability?` field 
notice that it is required
notice that the unit status field is no longer there on the unit form 
notice that the `What is the listing availability?` field's data shows up correctly on the listing detail page on the partner site

## Checklist:


## Reviewer Notes:

Steps to review a PR:

- Read and understand the issue, and ensure the author has added QA notes
- Review the code itself from a style point of view
- Pull the changes down locally and test that the acceptance criteria is met
- Also review the acceptance criteria on the Netlify deploy preview (noting that these do not yet include any backend changes made in the PR)
- Either explicitly ask a clarifying question, request changes, or approve the PR if there are small remaining changes but the PR is otherwise good to go

## On Merge:

If you have one commit and message, squash. If you need each message to be applied, rebase and merge.
